benchmarks/ssr/common.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+const compiler = require('../../dist/compiler.common.js')
+const self = (global || root)
+
+self.performance = {
+  now: function () {
+    var hrtime = process.hrtime()
+    return ((hrtime[0] * 1000000 + hrtime[1] / 1000) / 1000)
+  }
+}
+
+function generateGrid (rowCount, columnCount) {
+  var grid = []
+
+  for (var r = 0; r < rowCount; r++) {
+    var row = { id: r, items: [] }
+    for (var c = 0; c < columnCount; c++) {
+      row.items.push({ id: (r + '-' + c) })
+    }
+    grid.push(row)
+  }
+
+  return grid
+}
+
+const gridData = generateGrid(10000, 100)
+
+var perfMixin = {
+  computed: {
+    performance: {
+      cached: false,
+      get: function () {
+        return (self.performance.now() - self.s).toFixed(2)
+      }
+    }
+  }
+}
+
+var gridComponent = {
+  template: '<div><h1>{{ performance }}ms</h1><my-table></my-table></div>',
+  mixins: [perfMixin],
+  components: {
+    myTable: {
+      data: function () {
+        return {
+          grid: gridData
+        }
+      },
+      template: '<table width="100%" cellspacing="2"><row v-for="row in grid" :row="row"></row></table>',
+      components: {
+        row: {
+          props: ['row'],
+          mixins: [perfMixin],
+          template: '<tr><th>{{ performance }}ms</th><column v-for="item in row"></column></tr>',
```

Comment:
Here it should be `v-for="item in row.items"`... previous versions were ignoring the columnCount all together. This explains why numbers were off - we should probably just test 1000 x 10 which is a more practical scenario (Note it's 10k components, not just cells)